styleguide/source/_patterns/01-molecules/05-navigation/08-ribbon-user-menu/_ribbon-user-menu.scss

```diff
@@ -36,18 +37,17 @@
   right: 0;
   z-index: 100;
   background-color: $black-20;
-  box-shadow: 3px 3px 2px $black-55;
+  box-shadow: -3px 3px 2px $black-55;
 
-  &.is-active {
-    display: block;
-  }
+  &.is-active {}
 
   .ribbon_user-menu_nav_item.divider_item {
     display: none;
   }
 
   @include breakpoint(740px) {
-    display: block;
+    display: block !important;
+    height: auto !important;
```

Comment:
@aCyborg I tried a viewport check in js but it seemed messy and not super reliable.